src/Symfony/Component/AssetMapper/ImportMap/ImportMapRenderer.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AssetMapper\ImportMap;
+
+/**
+ * @experimental
+ *
+ * @author Kévin Dunglas <kevin@dunglas.dev>
+ * @author Ryan Weaver <ryan@symfonycasts.com>
+ */
+class ImportMapRenderer
+{
+    public function __construct(
+        private readonly ImportMapManager $importMapManager,
+        private readonly string $charset = 'UTF-8',
+        private readonly string|false $polyfillUrl = ImportMapManager::POLYFILL_URL,
+        private readonly array $scriptAttributes = [],
+    ) {
+    }
+
+    public function render(?string $entryPoint = null): string
+    {
+        $attributeString = '';
+        foreach ($this->scriptAttributes as $name => $value) {
+            $attributeString .= ' ';
+            if ($value === true) {
+                $attributeString .= $name;
+
+                continue;
+            }
+            $attributeString .= sprintf('%s="%s"', $name, $this->escapeAttributeValue($value));
```

Comment:
shouldn't we also escape names ? We have no idea whether they come from a trusted source of no (and this requires a stricter escaping than values as they are not inside quotes. See Twig's `html_attr` escaping for the logic)

Reply:
To me, these ARE trusted values. They're passed in through the constructor. And in practice, they are configured in a YAML file. Do we really think this is a possible security vector where these are controlled by an untrusted source?